staging/src/k8s.io/apimachinery/pkg/runtime/serializer/cbor/cbor.go

```diff
@@ -79,6 +80,15 @@ func Strict(s bool) Option {
 	}
 }
 
+// Transcode configures a serializer to transcode the "raw" bytes of a decoded runtime.RawExtension
+// or metav1.FieldsV1 object to JSON. This is enabled by default to support existing programs that
+// depend on the assumption that objects of either type contain valid JSON.
+func Transcode(s bool) Option {
```

Comment:
This godoc is great. Could we get similar godoc on `Strict` and `NewSerializer` so it's clear what all the defaults are?